core/jazz_t-vault/test/test.js

```diff
@@ -15,13 +15,3519 @@
+          errorType: 'inputError',
+          message: 'Input cannot be empty'
+        });
+      });
+    done();
+  });
+
+  it('should reject while validating create safe input params with out event.body', (done) => {
+    delete event.body;
+    validations.validateCreateSafeInput(event)
+      .catch((err) => {
+        expect(err).to.include({
+          errorType: 'inputError',
+          message: 'Input cannot be empty'
+        });
+      });
+    done();
+  });
+
+  it('should reject while validating create safe input params with out required fields', (done) => {
+    delete event.body.name;
+    validations.validateCreateSafeInput(event)
+      .catch((err) => {
+        expect(err).to.include({
+          errorType: 'inputError',
+          message: 'Following field(s) are required - name'
+        });
+      });
+    done();
+  });
+
+  it('should resolve while validating update safe input params with valid input', (done) => {
+    validations.validateUpdateSafeInput(event)
+      .then((result) => {
+        assert(true);
+      })
+    done();
+  });
+
+  it('should reject while validating update safe input params with empty input', (done) => {
+    event.body = {};
+    validations.validateUpdateSafeInput(event)
+      .catch((err) => {
+        expect(err).to.include({
+          errorType: 'inputError',
+          message: 'Input cannot be empty'
+        });
+      });
+    done();
+  });
+
+  it('should reject while validating update safe input params with out event.body', (done) => {
+    delete event.body;
+    validations.validateUpdateSafeInput(event)
+      .catch((err) => {
+        expect(err).to.include({
+          errorType: 'inputError',
+          message: 'Input cannot be empty'
+        });
+      });
+    done();
+  });
+
+  it('should resolve while validating valid safename as path param', (done) => {
+    event.path = { "safename": "testsafe" }
+    validations.validateSafeInput(event)
+      .then((result) => {
+        assert(true);
+      })
+    done();
+  });
+
+  it('should reject while validating empty path param', (done) => {
+    event.path = {};
+    validations.validateSafeInput(event)
+      .catch((err) => {
+        expect(err).to.include({
+          errorType: 'inputError',
+          message: 'Input path cannot be empty'
+        });
+      });
+    done();
+  });
+
+  it('should reject while validating with out event.path', (done) => {
+    validations.validateSafeInput(event)
+      .catch((err) => {
+        expect(err).to.include({
+          errorType: 'inputError',
+          message: 'Input path cannot be empty'
+        });
+      });
+    done();
+  });
+
+  it('should reject while validating path params with out safename', (done) => {
+    event.path = { "name": "test" };
+    validations.validateSafeInput(event)
+      .catch((err) => {
+        expect(err).to.include({
+          errorType: 'inputError',
+          message: 'Following field(s) are required in path- safename'
```

Comment:
Can we keep this message and format consistent? `Following field(s) are required in path - ` Adding space before and after `-`

Reply:
Done